### DISCUSSION TITLE
dotnet tool

### DISCUSSION THREAD

**Discussion by apex944** [mivano/azure-cost-cli#167] [Q&A] [answered]
OK I have spent a few hours researching why I am unable to execute the first step of installing this awesome looking tool.  
I realize this is not a dotnet help discussion group but given the number of threads I just read had no mention of others having problems with "dotnet tool ..."  I thought I'd give it a shot here.  
So I'm running the .NETCore.App 8.0.7 run time in MacOS 
I get an error when I attempt to execute 'dotnet tool install' (and many other commands) that command could not be loaded, possible because "tool" does not exist or no SDKs were found.
I installed the Dotnet runtime but did not install any .NET SDKs.   Are they required?

**Comment by mivano** [comment, marked as answer]
Yes, you do need the SDK to use the dotnet tool command. See https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools where they mention it applies to SDK. 
I will make it more clear in the reader.

**Reply by apex944** [reply]
uninstalling runtime and then re-installing w/ SDK worked.  Loving this tool !

**Reply by mivano** [reply]
Great! Enjoy the tool.
